src/routes/routes.py

```diff
@@ -0,0 +1,98 @@
+import heapq as hq
+from typing import Dict, List, Tuple
```

Comment:
These have been [deprecated](https://docs.python.org/3.13/library/typing.html#typing.Dict) since 3.9. You can just use the builtin `dict`, `list`, and `tuple` now. Doesn't really matter, but it's just nice I guess.